stdlib/public/stubs/LibcShims.cpp

```diff
@@ -18,8 +18,17 @@
 #include <string.h>
 #include "../SwiftShims/LibcShims.h"
 
+#ifdef __ANDROID__
+extern "C" {
+extern size_t dlmalloc_usable_size(void*);
+// arc4random_random is missing in Android headers, but does exist.
+extern unsigned int arc4random_uniform(unsigned int upper_bound);
+}
+#else
+// Android's ssize_t is an int, which is the same size as long int.
```

Comment:
Please adjust the definition of `__swift_ssize_t` instead of commenting out the sanity check.

This is not the first time people misunderstand the purpose of this `static_assert`, can you think of a better way to word the error message below?